deploy.php

```diff
@@ -0,0 +1,459 @@
+<?php
+
+namespace Deployer;
+
+use Symfony\Component\Yaml\Yaml;
+
+require 'recipe/symfony3.php';
+require 'recipe/cachetool.php';
+require 'recipe/sentry.php';
+
+// Define some variables
+set('client', '$client');
+set('project', '$project');
+set('repository', '$repository');
+set('release', 'current');
+set('production_url', '$productionUrl');
+set('sentry_organization', '$sentryOrganization');
+set('sentry_project_slug', '$sentryProjectSlug');
+set('sentry_token', '$sentryToken');
+
+// Define staging
+host('dev03.sumocoders.eu')
+    ->user('sites')
+    ->stage('staging')
+    ->set('deploy_path', '~/apps/{{client}}/{{project}}')
+    ->set('branch', 'staging')
+    ->set('bin/php', 'php7.4')
+    ->set('bin/composer', '{{bin/php}} /home/sites/apps/{{client}}/{{project}}/shared/composer.phar')
+    ->set('cachetool', '/var/run/php_74_fpm_sites.sock')
+    ->set('document_root', '~/php74/{{client}}/{{project}}');
+
+// Define production
+//host('$host')
+//    ->user('sites')
```

Comment:
Misschien ook beter `$user` gebruiken ipv `sites`. Dan is het duidelijker dat dit moet ingevuld worden.

Reply:
Done in https://github.com/sumocoders/forkcms/pull/249/commits/3cb9801181264b962574c4b92391eadf4856a6e8